### DISCUSSION TITLE
Adding clear feature

### DISCUSSION THREAD

**Discussion by NigelPirrung** [jagrosh/MusicBot#782] [Ideas]
Is there any way to add a clear or purge command that removes all of the mentions and responses from the bot in the text channel?

**Comment by bjarkeh14** [comment]
I'll love to have this as well, So the bot can do a auto cleanup of all the text regarding the bot.

An example, that it will remove the command typed, and the bots own response after 5 or 10 sec

**Comment by MWE001** [comment]
I use the inter-punct bot for this very thing. However a native feature would be real nice to avoid having to install yet another bot.
